docs/reference/ml/trained-models/apis/get-trained-models-stats.asciidoc

```diff
@@ -127,14 +135,31 @@ The deployment stats for each node that currently has the model allocated.
 (double)
 The average time for each inference call to complete on this node.
 
+`error_count`:::
+(integer)
+The number of errors when evaluating the trained model.
+
+
 `inference_count`:::
 (integer)
 The total number of inference calls made against this node for this model.
 
+`inference_threads`:::
+(integer)
+The number of threads used by the inference process.
+This figure may be lower than top level value if there is not sufficient
+hardware threads to satisfy the configured value.
+
 `last_access`:::
 (long)
 The epoch time stamp of the last inference call for the model on this node.
 
+`model_threads`:::
+(integer)
+The number of threads used when sending inference requests to the model.
+This figure may be lower than top level value if there is not sufficient
```

Comment:
>  top level value if there is not sufficient ... configured value

Ditto re if this is the "model_threads" option on the start deployment API, can we state that explicitly?